### PR TITLE
Simple tooltip component

### PR DESCRIPTION
**Pull request by Golodhros**
Adding a simple Tooltip component

NOTE: I haven't addressed the integration with the stacked area, another PR will do that

### CODE REVIEW

**Comment by jchen-eb** [comment]
Nice, LGTM!

**Comment by jchen-eb** [comment]
needs rebase

File: src/charts/tooltip/tooltipComponent.js [hunk 1]
```diff
@@ -1 +1,81 @@
...
+    render() {
+        return (
+            <div className="tooltip-container" ref={this._setRef.bind(this)} />
+        );
+    }
+
+}
+
+
+// dateFormat string
```
Comment (rwholey-eb) [inline src/charts/tooltip/tooltipComponent.js]: curious what is this here?

Comment (Golodhros) [reply src/charts/tooltip/tooltipComponent.js]: Good catch! Cleaning.

**Pull request merged**